spec/README.md

```diff
@@ -79,64 +79,59 @@ A reference to a _term_ looks like this.
 > The provisions of the stability policy are not in effect until
 > the conclusion of the technical preview and adoption of this specification.
 
-Updates to this specification will not change
-the syntactical meaning, the runtime output, or other behaviour
-of valid messages written for earlier versions of this specification
-that only use functions defined in this specification.
+Updates to this specification will not make any valid _message_ invalid.
+
 Updates to this specification will not remove any syntax provided in this version.
-Future versions MAY add additional structure or meaning to existing syntax.
+
```

Comment:
@aphillips This change would satisfy my remaining concerns.

```suggestion

Updates to this specification will not introduce new runtime error conditions that would
produce errors or require the use of fallback values when formatting messages
that formatted without error or fallback with an earlier version of this specification.

```

Reply:
Thanks for this. I think it could need tweaking?

```suggestion
Updates to this specification MUST NOT specify an error for any message
that previously did not specify an error.

Updates to this specification MUST NOT specify the use of a fallback value for any message
that previously did not specify a fallback value.
```

We need to guard against new error conditions (your text) but also assignment of existing errors to new criteria. Also, note that I focus on specifying, which is something _we_ control. Messages don't generate errors/fallbacks: implementations do. Non-conformant implementations should not constrain us.

Reply:
Sure, that works as well. I edited your suggestion to add in the leading and trailing empty lines; otherwise these will connect with other paragraphs.